tests/test_security.py

```diff
@@ -88,8 +90,11 @@ def test_invalid_filter_attr():
 
 def test_invalid_tag_char():
     assert (
-        clean('<script/xss src="http://xx.com/xss.js"></script>') ==
-        '&lt;script xss="" src="http://xx.com/xss.js"&gt;&lt;/script&gt;'
+        clean('<script/xss src="http://xx.com/xss.js"></script>') in
+        [
+            '&lt;script src="http://xx.com/xss.js" xss=""&gt;&lt;/script&gt;',
+            '&lt;script xss="" src="http://xx.com/xss.js"&gt;&lt;/script&gt;'
```

Comment:
I think to fix this we need to sort the attributes and then escape it. That happens in `.disallowed_token` in the sanitizer. I'll think about that.